crates/yulgen/src/mappers/expressions.rs

```diff
@@ -25,6 +25,7 @@ use yultsur::*;
 pub fn expr(context: &mut FnContext, exp: &Node<fe::Expr>) -> yul::Expression {
     let expression = match &exp.kind {
         fe::Expr::Name(_) => expr_name(exp),
+        fe::Expr::Path(_) => todo!(), // XXX
```

Comment:
dont think there's a case where we generate Yul from a path. constants are lowered away